### DISCUSSION TITLE
Can Cypress save current datetime to use as a variable?

### DISCUSSION THREAD

**Discussion by paulappl** [cypress-io/cypress#16623] [Questions and Help]
Hi, 

I'm writing tests for a word processor which has a save function. When the save is invoked, a status bar updates with the datetime of the last save. Clicking the button and triggering the save is fine, but I'd like to verify that the status bar has updated and it seems the logical method would be to record the time at the start of the test (as InitialDatetime for example) at the start, trigger the save and verify StatusBarDatetime > InitialDatetime.

I thought I'd found a few clues here and there so have tried various ideas including 

`const todaysDate = Cypress.moment().format('MMM DD, YYYY')`

and saving as a variable, or 

`timestamp = Cypress.moment().utc();`
`cy.log('${timestamp}')`

but Cypress.moment seems to throw an error (I might be using it incorrectly, or even in the wrong place) and I don't feel I'm getting any closer. I'm very new to this stuff so apologies if there is an obvious solution! 

thanks,
Paul

**Comment by paulappl** [comment]
Update - I see that moment is considered old. Have started looking at daysjs instead but am struggling with how to use it!

**Reply by jennifer-shehane** [reply]
There's an example on how to use dayjs in your tests here: https://github.com/cypress-io/cypress-example-recipes/tree/master/examples/blogs__dayjs

**Reply by paulappl** [reply]
Thank you @jennifer-shehane

**Comment by julikoff0** [comment]
I tried Cypress.moment() too. But he answers with an error. And I found a good solution. It looks like this:
`const dateNow = new date().toLocaleDateString()`
